### ISSUE TITLE
Compiler Advice

### ISSUE THREAD

**Issue by g000001** [guicho271828/common-lisp-extensions#43]
Feature description:

Supported implementations:
+ [x] [SBCL]()
+ [ ] [CCL]()
+ [ ] [ECL]()
+ [x] [CMU]()
+ [ ] [ABCL]()
+ [ ] [ALISP]()
+ [x] [LispWorks]()
    - [LispWorks: deftransform](http://www.lispworks.com/documentation/lw71/LW/html/lw-435.htm)

## SBCL / CMUCL

`deftransform`

## LispWorks

`deftransform`

**Comment by guicho271828** [comment]
Is the API compatible between LW and Python? Also, if the link to the document is available, please add it to the link in the bullet box description.

**Comment by g000001** [comment]
> Is the API compatible between LW and Python?

No. LispWorks' one is simple source code transform much like the `define-compiler-macro`.
SBCL/CMUCL's one is more sophisticated.
